sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -204,6 +201,17 @@ def _get_severity_level(severity_number: Optional[SeverityNumber]):
         return 0
     return int((severity_number.value - 1) / 4 - 1)
 
+def _body_to_string(log_body: Any) -> str:
+    if not log_body:
+        return "n/a"
+
+    if isinstance(log_body, str):
+        return log_body[:32768]
+
+    if isinstance(log_body, Exception):
+        return str(log_body)[:32768]
+
+    return json.dumps(log_body)[:32768]
```

Comment:
We should probably handle the use case if the body is not serializable by catching the exception and using str() as the default.